### DISCUSSION TITLE
How can I update the full content in the doc

### DISCUSSION THREAD

**Discussion by mazhenjie233** [toeverything/blocksuite#7255] [Q&A]
The content of my document will be constantly updated because of the backend, which will return markdown content to me. I need to insert the latest markdown into the document, and the backend transmits it to the frontend in a streaming way, just like an AI answer, so it needs to be constantly updated. Update, what API should be used to solve the problem. thank you~

**Comment by raintoway** [comment]
use `provider` to subscribe websocket/sse ,and then use `applyupdate` to apply the streaming-update to doc.
